examples/middleman/Gemfile.lock

```diff
@@ -1,108 +1,113 @@
+    ffi (1.16.3)
+    haml (6.3.0)
+      temple (>= 0.8.2)
+      thor
       tilt
     hamster (3.0.0)
       concurrent-ruby (~> 1.0)
     hashie (3.6.0)
-    i18n (0.9.5)
+    i18n (1.6.0)
       concurrent-ruby (~> 1.0)
-    kramdown (2.3.0)
+    kramdown (2.4.0)
       rexml
-    listen (3.0.8)
-      rb-fsevent (~> 0.9, >= 0.9.4)
-      rb-inotify (~> 0.9, >= 0.9.7)
+    listen (3.9.0)
+      rb-fsevent (~> 0.10, >= 0.10.3)
+      rb-inotify (~> 0.9, >= 0.9.10)
     memoist (0.16.2)
-    middleman (4.3.11)
+    middleman (4.5.1)
       coffee-script (~> 2.2)
       haml (>= 4.0.5)
       kramdown (>= 2.3.0)
-      middleman-cli (= 4.3.11)
-      middleman-core (= 4.3.11)
-    middleman-autoprefixer (2.10.1)
-      autoprefixer-rails (~> 9.1)
-      middleman-core (>= 3.3.3)
-    middleman-cli (4.3.11)
-      thor (>= 0.17.0, < 2.0)
-    middleman-core (4.3.11)
-      activesupport (>= 4.2, < 6.0)
-      addressable (~> 2.3)
+      middleman-cli (= 4.5.1)
+      middleman-core (= 4.5.1)
+    middleman-autoprefixer (3.0.0)
+      autoprefixer-rails (~> 10.0)
+      middleman-core (>= 4.0.0)
+    middleman-cli (4.5.1)
+      thor (>= 0.17.0, < 1.3.0)
+    middleman-core (4.5.1)
+      activesupport (>= 6.1, < 7.1)
+      addressable (~> 2.4)
       backports (~> 3.6)
-      bundler
-      contracts (~> 0.13.0)
+      bundler (~> 2.0)
+      contracts (~> 0.13, < 0.17)
       dotenv
       erubis
       execjs (~> 2.0)
       fast_blank
       fastimage (~> 2.0)
       hamster (~> 3.0)
       hashie (~> 3.4)
-      i18n (~> 0.9.0)
-      listen (~> 3.0.0)
+      i18n (~> 1.6.0)
+      listen (~> 3.0)
       memoist (~> 0.14)
-      padrino-helpers (~> 0.13.0)
+      padrino-helpers (~> 0.15.0)
       parallel
       rack (>= 1.4.5, < 3)
       sassc (~> 2.0)
       servolux
       tilt (~> 2.0.9)
+      toml
       uglifier (~> 3.0)
-    minitest (5.14.3)
-    padrino-helpers (0.13.3.4)
-      i18n (~> 0.6, >= 0.6.7)
-      padrino-support (= 0.13.3.4)
+      webrick
+    minitest (5.22.2)
+    padrino-helpers (0.15.3)
+      i18n (>= 0.6.7, < 2)
+      padrino-support (= 0.15.3)
       tilt (>= 1.4.1, < 3)
-    padrino-support (0.13.3.4)
-      activesupport (>= 3.1)
-    parallel (1.20.1)
-    public_suffix (4.0.6)
-    rack (2.2.3)
-    rb-fsevent (0.10.4)
+    padrino-support (0.15.3)
+    parallel (1.24.0)
+    parslet (2.0.0)
+    public_suffix (5.0.4)
+    rack (2.2.8.1)
+    rb-fsevent (0.11.2)
     rb-inotify (0.10.1)
       ffi (~> 1.0)
-    rexml (3.2.4)
+    rexml (3.2.6)
     sassc (2.4.0)
       ffi (~> 1.9)
     servolux (0.13.0)
-    temple (0.8.2)
-    thor (1.1.0)
-    thread_safe (0.3.6)
-    tilt (2.0.10)
-    tzinfo (1.2.9)
-      thread_safe (~> 0.1)
+    temple (0.10.3)
+    thor (1.2.2)
+    tilt (2.0.11)
+    toml (0.3.0)
+      parslet (>= 1.8.0, < 3.0.0)
+    tzinfo (2.0.6)
+      concurrent-ruby (~> 1.0)
     uglifier (3.2.0)
       execjs (>= 0.3.0, < 3)
+    webrick (1.8.1)
 
 PLATFORMS
   x86_64-linux
```

Comment:
Why is this one the same when the other one was changed?

Reply:
See [this comment](https://github.com/vercel/vercel/pull/11203#discussion_r1507777893). In this case I'd guess there is still a dependency that has a `x86_64-linux` pre-compiled binary, but that binary comes pre-installed on MacOS (where I ran `bundle`).